src/tests/test_inspect.py

```diff
@@ -127,3 +131,61 @@ def test_get_schema_names(ip):
     schema_names = inspect.get_schema_names()
     for schema in schema_names:
         assert schema in expected_schema_names
+
+
+@pytest.mark.parametrize(
+    "get_columns, rows, field_names",
+    [
```

Comment:
I disagree. The purpose of using parametrized tests is to reduce the boilerplate code. these are all testing the same function under different scenarios. splitting them would cause a lot of duplicated code.